tests/protractor/e2e/medic-collect.js

```diff
@@ -0,0 +1,76 @@
+const assert = require('chai').assert,
+    net = require('net'),
+    host = 'localhost',
+    port = '5998',
+    dbName = 'medic';
+
+describe('medic-collect', () => {
+
+  describe('without User-Agent header', () => {
+
+    it('is prompted for auth details if not supplied', () => {
+
+      // when
+      return rawHttpRequest(
+
+`HEAD /${dbName}/_design/medic/_rewrite/add?deviceID=imei%3A357578064823168 HTTP/1.1
+X-OpenRosa-Version: 1.0
+Date: Tue, 11 Apr 2017 06:34:21 CEST
+Host: ${host}:${port}
+Connection: close
+
+`).then((res) => {
+
+        // then
+        assert.equal(res.statusCode, 401, JSON.stringify(res));
+        assert.equal(res.headers['WWW-Authenticate'], 'Basic realm="Medic Mobile Web Services"', JSON.stringify(res));
+
+      });
+
+    });
+
+    it('can request a list of forms', (done) => {
+      done('TODO');
```

Comment:
IMO remove the TODO tests. I agree it's nice to have but TODOs just add clutter here.